go/vm/lfvm/opcode.go

```diff
@@ -226,6 +228,7 @@ const (
 	NUM_OPCODES
 )
 
+// TODO: document order of this map initialization if any.
```

Comment:
There should be none. Since maps in Go are randomly ordered, the initialization order has no effect.